Tests/SwiftDocCTests/Semantics/DoxygenTests.swift

```diff
@@ -108,10 +110,10 @@ class DoxygenTests: XCTestCase {
         XCTAssertEqual(renderNode.primaryContentSections.count, 1)
 
         let overviewSection = try XCTUnwrap(renderNode.primaryContentSections.first as? ContentRenderSection)
-        XCTAssertEqual(overviewSection.content.count, 3)
+        XCTAssertEqual(overviewSection.content.count, 4)
         XCTAssertEqual(overviewSection.content, [
             .heading(.init(level: 2, text: "Overview", anchor: "overview")),
-
+            .paragraph(.init(inlineContent: [.text("This is description with abstract.")])),
```

Comment:
Should we try to actually parse `@abstract` as an abstract? I feel like it would stay true to the use of the command.

On the other hand, we *probably* shouldn't add too much Doxygen/HeaderDoc support, since we really want to push people to using Markdown... 😅 I'm fine either way on this, though i'm curious what the workgroup (or just the rest of our team) thinks.

Reply:
With the thought of having limited Oxygen/HeaderDoc support I thought we should not restrain from parsing @abstract as an abstract. But open to other ideas too and can be convinced otherwise.

Reply:
I feel like we should at least explore what it would take for `@abstract` to become the abstract instead of being a sentence in the discussion. 

It might also be good to verify that links in the abstract are resolved. IIRC this doesn't happen automatically for new elements.